focuspoints.lrdevplugin/FocusPoint.lua

```diff
@@ -14,27 +14,37 @@
   limitations under the License.
 --]]
 
-local LrSystemInfo = import 'LrSystemInfo'
 local LrFunctionContext = import 'LrFunctionContext'
 local LrApplication = import 'LrApplication'
 local LrDialogs = import 'LrDialogs'
-local LrView = import 'LrView'
```

Comment:
FYI, this is the exact reason I suggest the AppleDelegates require it's own imports. With this refactoring, it's possible it broke other delegates.